cocos/2d/CCNode.h

```diff
@@ -1862,6 +1847,19 @@ class CC_DLL Node : public Ref
     std::function<void()> _onEnterTransitionDidFinishCallback;
     std::function<void()> _onExitTransitionDidStartCallback;
 
+//Physics:remaining backwardly compatible  
+#if CC_USE_PHYSICS
+    PhysicsBody* _physicsBody;
+public:
+    void setPhysicsBody(Component* physicsBody) 
+    { 
+        addComponent(physicsBody);
+    }
+    PhysicsBody* getPhysicsBody() const { return _physicsBody; }
+
+    friend class PhysicsBody;
+#endif
```

Comment:
Please don't keep these functions. It is strange that Node can get physicsbody from component and from `getPhysicsBody()`.